tests/node_pool_deployer.py

```diff
@@ -70,18 +70,19 @@ def calculate_cpu_and_mem_per_node(self):
                  cpu_per_node: how many CPUs per node, must divide by 2 according to GCP specifications limits
                  mem per node: how much memory per node, must be withing GCP specifications limits
         """
+        # GCP limit
         max_cpus_per_node = 96
-        extra_cpus = 4
+        # extra cpus for k8s processes
+        extra_cpus_per_machine = 2
         total_cpu, _ = self.get_total_cpu_and_mem()
-        total_cpu += extra_cpus
         dividor = 6
         for i in range(1, 6):
-            if total_cpu / i > max_cpus_per_node:
+            if total_cpu / i > max_cpus_per_node + extra_cpus_per_machine:
                 continue
             dividor = i
             break
 
-        cpu_per_node = ceil(total_cpu / dividor)
+        cpu_per_node = ceil(total_cpu / dividor) + extra_cpus_per_machine
```

Comment:
The value here doesn't match the name of the variable - please update one of them to match the other.

What's this variable used for? It seems to me that leaving this line as it was before makes the most sense.

Reply:
fixed, and also found a bug there with the calculation, thanks